conan/tools/files/files.py

```diff
@@ -442,6 +442,7 @@ def untargz(filename, destination=".", pattern=None, strip_root=False, extract_f
         if not pattern and not strip_root:
             tarredgzippedFile.extractall(destination)
         else:
+            using_long_path_prefix = destination.startswith("\\\\?\\")
```

Comment:
```suggestion
            using_long_path_prefix = destination.startswith(r"\\?\")
```

Reply:
I tried that at first, but it raised a syntax error:

```
SyntaxError: unterminated string literal (detected at line 445); perhaps you escaped the end quote?
```

So it should be `r"\\?\\"` instead.  That's why I decided to use this one 🤷‍♂️ 

```suggestion
            using_long_path_prefix = destination.startswith(r"\\?\\")
```
Do you prefer this one?

Reply:
I don't get it. If using the ``r``, I thought it was possible to use the exact literal string that @AbrilRBS proposed